BondageClub/Scripts/Skill.js

```diff
@@ -45,7 +45,9 @@ function SkillChange(SkillType, SkillLevel, SkillProgress, Push) {
 	}
 
 	else {
-		throw new Error("trying to change invalid skill");
+		if (CurrentScreen != "Login") {
+			throw new Error("trying to change invalid skill");
```

Comment:
```suggestion
			console.warn(`Invalid skill type "${SkillType}"`);
```
Same comment here as in `Reputation.js`.